packages/stark-core/base.spec.ts

```diff
@@ -1,17 +1,16 @@
 "use strict";
 
-import "core-js/es6";
-import "core-js/es7/reflect";
-import "core-js/stage/4";
+import "../../node_modules/core-js/es";
+import "../../node_modules/core-js/proposals/reflect-metadata";
 
 /* tslint:disable:no-import-side-effect */
-import "zone.js/dist/zone";
-import "zone.js/dist/long-stack-trace-zone";
-import "zone.js/dist/proxy"; // since zone.js 0.6.15
-import "zone.js/dist/sync-test";
-import "zone.js/dist/jasmine-patch"; // put here since zone.js 0.6.14
-import "zone.js/dist/async-test";
-import "zone.js/dist/fake-async-test";
+import "../../node_modules/zone.js/dist/zone";
+import "../../node_modules/zone.js/dist/long-stack-trace-zone";
+import "../../node_modules/zone.js/dist/proxy"; // since zone.js 0.6.15
+import "../../node_modules/zone.js/dist/sync-test";
+import "../../node_modules/zone.js/dist/jasmine-patch"; // put here since zone.js 0.6.14
+import "../../node_modules/zone.js/dist/async-test";
+import "../../node_modules/zone.js/dist/fake-async-test";
```

Comment:
Could you add a FIXME comment here? you can reference angular/angular#29531

Reply:
Same response: 

> Well, in fact, this is not related to Angular...